main.py

```diff
@@ -0,0 +1,143 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+from models import Author, Book
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+# AUTHOR endpoints
+@app.post("/authors/", response_model=schemas.AuthorCreate)
```

Comment:
The response model for the `create_author` endpoint should be `schemas.Author`, not `schemas.AuthorCreate`, as it returns the created author object.